src/pages/getting-started/index.mdx

```diff
@@ -38,121 +62,106 @@ From a more tech perspective, the LiveChat Platform provides:
 
-Web appplications can be displayed **fullscreen**, right in the Agent App. If you're planning on integrating your CRM with LiveChat, that might be the way to go.
+To access basic information in LiveChat and HelpDesk accounts, you will use <a href="https://developers.livechat.com/docs/authorization/sign-in-with-livechat" target="_blank">Accounts SDK (Sign in with LiveChat)</a>, which is the simplest way to do that.
 
-There's also another area where developers can integrate their apps- it's the Message Box. This interface can be used for extensions related strongly to messages, which are being sent.
-It could be used to send gifs or files from <a href="https://www.livechat.com/marketplace/apps/dropbox/" target="_blank">Dropbox</a>.
+Platform also provides you with with a <a href="https://developers.livechat.com/docs/authorization/global-accounts" target="_blank">Global Accounts service</a> which is very useful for customers who use <a href="https://www.chatbot.com/" target="_blank">ChatBot</a>, <a href="https://www.helpdesk.com/" target="_blank">HelpDesk</a>, or <a href="https://www.knowledgebase.com/" target="_blank">KnowledgeBase</a>. It allows customers to configure standard options and resources across all products.
 
-<div align="center">
-  <iframe
-    width="560"
-    height="315"
-    src="https://www.youtube.com/embed/P07QVFOrT04"
-    frameBorder="0"
-    allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
-    allowFullScreen
-  ></iframe>
-</div>
+# Dveloper Console tools
 
-## Chat Widget
+If you want to save time while building your app and don't want to play with APIs, <Placeholder id="DEVELOPER_CONSOLE_URL"/> comes with a helping hand to provide easy access to all tools and resources for developers who want to build their apps. It is a place where you can configure your applications, set up their most crucial functions and create all the necessary extensions. The simplest way to create a paid app or display your app on the Marketplace is within the <Placeholder id="DEVELOPER_CONSOLE_URL"/>.
 
-Just as there are ways to build upon the Agent App, there are also some to extend the Chat Widget. First of all, you could use [Chat Widget JavaScript API](https://developers.livechat.com/docs/js-api/) to alter the look and behavior of the Chat Widget. If that's not enough, building a new Chat Widget from scratch is possible with the [Chat Widget Customer SDK](https://developers.livechat.com/docs/visitor-sdk/).
+<Placeholder id="DEVELOPER_CONSOLE_URL"/> is rich in many helpful functions that will help make your app efficient, valuable, and attractive for potential customers.
```

Comment:
This sentence sounds a bit like a marketing promise. I would either write another sentence or drop it